src/controllers/userAuth.ts

```diff
@@ -197,6 +197,50 @@ const userAuthController = {
                 }
             });
         }
+    },
+
+    resetPassword: async (req: Request, res: Response) => {
+        const errors = validationResult(req);
+
+        if (!errors.isEmpty()) {
+            res.status(statusCodes.MISSING_PARAMS).json(
+                {
+                    status: 422,
+                    message: `Error: there are missing parameters.`
+                }
+            );
+        } else {
+            const oldPassword = req.body.oldPassword;
+            const newPassword = req.body.newPassword;
+            const jwtToken = req.body.jwtToken;
```

Comment:
can we change from req.body.token to

`request.headers?.authorization.split(" ")[1]
`
i.e. get token from request headers just to keep it consistent with how we're getting it for other api's